<launch>
    <!-- PICK-UP ZONE -->
    <param name="/plemma/hsr/pickup_pose_x"       value="-5.0" />
    <param name="/plemma/hsr/pickup_pose_y"       value="-1.0" />
    <param name="/plemma/hsr/pickup_pose_yaw_deg" value="45" />
    <!-- DROP-OFF ZONE -->
    <param name="/plemma/hsr/dropoff_pose_x"       value="-10.0" />
    <param name="/plemma/hsr/dropoff_pose_y"       value="-2.0" />
    <param name="/plemma/hsr/dropoff_pose_yaw_deg" value="30" />
    <node pkg="pick_objects"       type="pick_objects_node"
          name="pick_objects" output="screen" />
</launch>
